lib/brakeman.rb

```diff
@@ -408,7 +408,11 @@ def self.write_report_to_formats tracker, output_formats
   private_class_method :write_report_to_formats
 
   def self.page_output text
-    if system("which less")
+    ci = ENV["CI"]
+
+    if ci.is_a? String and ci.downcase == "true"
```

Comment:
should be `$stdout.tty?` instead ?

Reply:
See https://github.com/presidentbeef/brakeman/blob/ab3fc23bfdef7f7ea22ba2fee8641592793263cf/lib/brakeman.rb#L400